src/spikeinterface/exporters/to_phy.py

```diff
@@ -64,6 +65,8 @@ def export_to_phy(
         Dtype to save binary data
     verbose: bool
         If True, output is verbose
+    use_relative_path : bool, default: False
+        If True saves the `dat_path` as a relative path, else an absolute
```

Comment:
```suggestion
        If True saves the binary file `dat_path` in the `params.py` relative to 'output_folder', if False, uses an absolute path
```

@h-mayorquin is this better? Or do you want something more substantial?

Reply:
This makes sense to me. Maybe you could add the actual example:

```python
        If True saves the binary file `dat_path` in the `params.py` relative to 'output_folder' (i.e. "looks like this", if False, uses an absolute path (i.,e. "looks like this").
```

Reply:
Also, maybe mention how it interats with `copy_binary` as it was described above?

Reply:
```suggestion
        If True saves the binary file `dat_path` in the `params.py` relative to 'output_folder' (ie `dat_path=r'recording.dat'`)
        If False, uses an absolute path in the `params.py` (ie `dat_path=r'path/to/the/recording.dat'`)
```

Reply:
Missed your last comment. Will add now.

Reply:
```suggestion
        If True and `copy_binary=True` saves the binary file `dat_path` in the `params.py` relative to `output_folder` (ie `dat_path=r'recording.dat'`). If `copy_binary=False`, then uses a path relative to the `output_folder`
        If False, uses an absolute path in the `params.py` (ie `dat_path=r'path/to/the/recording.dat'`)
```

The only thing I'm not sure of is what a good (not complicated) path example for copy_binary=False, and use_relative=True. 
But how does this look for now @h-mayorquin?

Reply:
Yeah, looks better to me. I don't have ideas on how to provide an example path. Thanks for improving this.